content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,65 @@
+Title: A tour of modern frontend with React and Flux
+Date: 2015-08-18
+Short_summary: React and Flux have swept the frontend in the last year or so. Let's look at it.
+Category: Dev
+Authors: Vincent
+
+
+*Reading time: ~15 minutes.*
+
+In this article we are going to have a look at my preferred set of tools for writing React/Flux webapps.
+If you don't know React or Flux, please look at their respective websites: [React](http://facebook.github.io/react/), [Flux](https://facebook.github.io/flux/).
+<!-- PELICAN_END_SUMMARY -->
+
+
+## Tools of the trade
+The React community is embracing ES6 — and sometimes ES7 — through [Babel](http://babeljs.io/). I personally like Typescript but recently decided to go with Babel for a few of reasons:
+
+- a constant changes in libraries and their sheer number were making it very hard to keep up-to-date definitions
+- Typescript is still lacking support for some things, making some libraries like [ImmutableJS](http://facebook.github.io/immutable-js/) and its records very hard to use
+- getting tired of casting things to <any> because of the compiler not picking up the correct things
+
+I do very much like types though, so I will have a look at [Flow](http://flowtype.org/) when it fully [supports ES6](https://github.com/facebook/flow/issues/560).
+
+You might also see the term isomorphic (confusing) or more recently universal, meaning that it can be rendered on the server as well.
+
+
+### Dev dependencies
```

Comment:
dev Dependencies is intended, it's to match the devDependencies object in package.json.
I was actually thinking of just putting devDependencies without space